test/Github.test.js

```diff
@@ -0,0 +1,41 @@
+const {expect} = require("chai");
+
+const Api = require('../src/Github-api')
+
+let api;
+
+before( function () {
+    api = new Api();
+})
+
+describe("Github Tests", function() {
+    it("Can get user information by username", async function() {
+        const username = "yuliashi";
+        const response = await api.getUser(username);
+        console.log(response)  
+
+        expect(response).to.have.property("type", "User"); 
+        expect(response).to.have.property("id").that.is.a('number')
```

Comment:
This is good, but most importantly you need to make sure you're getting the user that you're requesting. I.e. the user that you're getting back has the same username `yuliashi`